traffic_ops/testing/api/v1/servers_test.go

```diff
@@ -118,7 +116,7 @@ func UpdateTestServers(t *testing.T) {
 	// Attempt Update - should fail
 	_, _, err = TOSession.UpdateServerByID(remoteServer.ID, remoteServer)
 	if err == nil {
-		t.Errorf("expected error when updating Server Type of a server assigned to DSes")
+		t.Fatalf("expected error when updating Server Type of a server assigned to DSes")
```

Comment:
This one doesn't really matter either way, because it's at the end. Just seems a bit unnecessary.